pasta_eln/GUI/project.py

```diff
@@ -200,6 +201,12 @@ def execute(self, command:list[Any]) -> None:
         if '-branch' in doc and len(doc['-branch'])>0 and 'path' in doc['-branch'][0]:
           oldPath = self.comm.backend.basePath/doc['-branch'][0]['path']
           newPath = self.comm.backend.basePath/('trash_'+doc['-branch'][0]['path'])
+          if newPath.exits():
+            ret = QMessageBox.critical(self, 'Warning', 'Old project data exists. Do you want to delete?', \
```

Comment:
When a project is created once and deleted, thereafter whenever I delete the project with the same name, the following popup is shown, why is this popup important to the user?

![image](https://github.com/PASTA-ELN/pasta-eln/assets/141253011/fd61333e-091a-44c6-b76a-26e42d1c73df)